svelte/headless/package.json

```diff
@@ -10,12 +10,41 @@
 		".": {
 			"types": "./dist/index.d.ts",
 			"svelte": "./dist/index.js"
+		},
+		"./components/*": {
+			"types": "./dist/components/*/*.d.ts",
+			"default": "./dist/components/*/*.js"
```

Comment:
I think we should be consistent.
Should we put "svelte" or "default" everywhere?